### PR TITLE
Sprint-Challenge Applied Javascript - Lambda Times

### PR DESCRIPTION
**Pull request by tmshkr**

### CODE REVIEW

**Comment by tmshkr** [comment]
[Live Site](https://tmshkr.github.io/Sprint-Challenge-Applied-Javascript/)

File: README.md [hunk 1]
```diff
@@ -30,41 +30,59 @@ Edit this document to include your answers after each question. Make sure to lea
...
 
+A NodeList can be converted into an array in order to access array methods, such as `map`, `filter`, and `reduce`. This can be achieved by using `from` method on the `Array` prototype:
+```
+const arr = Array.from(nodeList)
+```
+The NodeList interface includes `forEach`, so that it can be used without converting to an array, and children in a NodeList can also be accessed using bracket notation, e.g., `nodeList[0]`.
+
 5. What is a component?
 
+A UI component is a reusable piece of code, which can create and append an HTML element dynamically with JavaScript, rather than writing an element into the HTML file. This is useful when developing applications with many similar elements, so that they do not need to be manually created and code is kept DRY. Components are often used to dynamically create HTML elements from data retrieved from an API.
```
Comment (jskway) [inline README.md:57]: 💯

File: components/Cards/index.js [hunk 2]
```diff
@@ -17,3 +17,41 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+function Card(article, topic) {
+  const ce = document.createElement.bind(document);
+
```
Comment (jskway) [inline components/Cards/index.js:23]: Awesome shorthand :+1:

File: components/Cards/index.js [hunk 3]
```diff
@@ -17,3 +17,41 @@
...
+  return card;
+}
+
+const cardsContainer = document.querySelector(".cards-container");
+
+axios
+  .get("https://lambda-times-backend.herokuapp.com/articles")
+  .then(({ data }) => {
+    const { articles } = data;
```
Comment (jskway) [inline components/Cards/index.js:50]: Nice use of destructuring 👍

File: components/Carousel/Carousel.js [hunk 4]
```diff
@@ -17,3 +17,44 @@
...
+    img.dataset.index = i;
+  });
+  const rightButton = carousel.appendChild(ce("div"));
+
+  carousel.className = "carousel";
+  leftButton.className = "left-button";
+  rightButton.className = "right-button";
+
+  leftButton.onclick = function() {
+    current > 0 ? current-- : (current = images.length - 1);
```
Comment (jskway) [inline components/Carousel/Carousel.js:40]: Awesome job on this carousel and using the ternary operator

File: components/Tabs/index.js [hunk 5]
```diff
@@ -7,3 +7,33 @@
 //
 //  The tab component should look like this:
 //    <div class="tab">topic here</div>
+
+function Tab(topic) {
+  const ce = document.createElement.bind(document);
+
+  const tab = ce("div");
+  tab.innerText = topic;
+  topic = topic.match(/\w*/)[0];
```
Comment (jskway) [inline components/Tabs/index.js:16]: Nice use of regex

**Review by jskway — APPROVED** [review]

**Pull request merged**